api/app/Modules/Library/Http/Transformers/DraftLyricsTransformer.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Modules\Library\Http\Transformers;
+
+use App\Modules\Core\Transformers\Transformer;
+use App\Modules\Library\Models\DraftLyrics;
+use League\Fractal\Resource\Primitive;
+
+class DraftLyricsTransformer extends Transformer
+{
+    protected array $availableIncludes = ['related'];
+
+    public function toArray(DraftLyrics $draftLyrics): array
+    {
+        return [
+            'id' => $draftLyrics->id,
+            'trackId' => $draftLyrics->track_id,
+            'document' => $draftLyrics->document->toArray(),
+            'createdAt' => $this->dateTime($draftLyrics->created_at),
+            'updatedAt' => $this->dateTime($draftLyrics->updated_at)
+        ];
+    }
+
+    public function includeRelated(DraftLyrics $draftLyrics): Primitive
```

Comment:
The `related` include is meant to contain just a list of `true/false` values, indicating whether the entity has a related object of a given type. I assume you copied this idea from the `TrackTransformer`, which ends up with something like this:
```json
{ 
  "related": {
    "lyrics": false,
    "audio": true,
    "video": false,
  }
}
```

You probably want to add an include like `track` similar to how the `TrackTransformer` has an `includeAlbum` method.

Or you can just remove this and we can only add that if/when we need it. I'm in favor of this approach.

Reply:
yeah you are correct, I copied the TrackTransformer, thought it would be similar, happy to add includeTrack too, will make that change right away